test/e2e/network/service.go

```diff
@@ -319,9 +319,11 @@ func verifyServeHostnameServiceUp(c clientset.Interface, ns, host string, expect
 		return fmt.Sprintf("for i in $(seq 1 %d); do %s http://%s 2>&1 || true; echo; done",
 			50*len(expectedPods), wget, serviceIPPort)
 	}
-	commands := []func() string{
+
+	var commands []func() string
+	if host != "" {
```

Comment:
We still need verify host network connectivity, but this parameter is not necessary anymore, you can launch a pod with hostNetwork, and use `framework.RunHostCmd` to run command from this pod, reference:
https://github.com/kubernetes/kubernetes/blob/master/test/e2e/network/service.go#L2007

Reply:
done.